packages/integration-sdk-core/src/data/converters.ts

```diff
@@ -169,7 +169,7 @@ export function convertNameValuePairs(
  * @see parseTimePropertyValue
  */
 export function getTime(
-  time: Date | string | number | undefined | null,
+  time: Date | string | undefined | null,
```

Comment:
Another breaking change. I feel great about this b/c:
1. This method is already deprecated
2. And instead of enhancing it we are removing functionality